src/System.Text.Json/experiments/Json.Net.Test/Json.Net.Tests/JsonNetComparison.cs

```diff
@@ -0,0 +1,229 @@
+            writer.WritePropertyName("Drives");
+            writer.WriteStartArray();
+            writer.WriteValue("DVD read/writer");
+            writer.WriteValue("500 gigabyte hard drive");
+            writer.WriteValue("200 gigabype hard drive");
+            writer.WriteEnd();
+            writer.WriteEndObject();
+            if (output) Console.WriteLine(sw.ToString());
+        }
+
+        private static void RunWriterTestJson()
+        {
+            Timer.Restart();
+            for (var i = 0; i < NumberOfIterations*NumberOfIterations; i++)
+            {
+                JsonWriterHelper(OutputJsonData);
+            }
+            TimingResultsJsonReader.Add(Timer.ElapsedMilliseconds);
+        }
+
+        private static void JsonWriterHelper(bool output)
+        {
+            var buffer = new byte[1024];
+            var stream = new MemoryStream(buffer);
+            var writer = new JsonWriter(stream, FormattingData.Encoding.Utf8, prettyPrint: true);
+            writer.WriteObjectStart();
+            writer.WriteAttribute("CPU", "Intel");
+            writer.WriteAttribute("PSU", "500W");
+            writer.WriteMember("Drives");
+            writer.WriteArrayStart();
+            writer.WriteString("DVD read/writer");
+            writer.WriteString("500 gigabyte hard drive");
+            writer.WriteString("200 gigabype hard drive");
+            writer.WriteArrayEnd();
+            writer.WriteObjectEnd();
+            if (output) Console.WriteLine(Encoding.UTF8.GetString(buffer, 0, (int)stream.Position));
+        }
+
+        private static void RunReaderTest()
+        {
+            Output("====== TEST Read ProjectLockJson ======");
+            for (var i = 0; i < NumberOfSamples; i++)
+            {
+                JsonNetReaderHelper(new StringReader(TestJson.ProjectLockJson), OutputJsonData);
+                // Do not test first iteration
+                RunTestJsonNet(TestJson.ProjectLockJson, OutputJsonData);
+                JsonReaderHelper(new Utf8String(TestJson.ProjectLockJson), OutputJsonData); // Do not test first iteration
+                RunTestJsonReader(TestJson.ProjectLockJson, OutputJsonData);
+            }
+
+            Output("Json.NET Timing Results");
+            foreach (var res in TimingResultsJsonNet)
+            {
+                Output(res.ToString());
+            }
+
+            Output("System.Text.Json Timing Results");
+            foreach (var res in TimingResultsJsonReader)
+            {
+                Output(res.ToString());
+            }
+            
+            TimingResultsJsonNet.Clear();
+            TimingResultsJsonReader.Clear();
+        }
+
+        private static void RunTestJsonNet(string str, bool output)
+        {
+            var utf8Str = new Utf8String(str);
+            Timer.Restart();
+            for (var i = 0; i < NumberOfIterations; i++)
+            {
+                JsonNetReaderHelper(new StringReader(utf8Str.ToString()), output);
```

Comment:
You're including the overhead of converting Utf8String to a string with Json.NET. Just pass it str.